scripts/check-version.sh

```diff
@@ -52,16 +52,16 @@ PUBLISHED_VERSIONS=$(git tag | grep "^v[0-9]\+\.[0-9]\+\.[0-9]\+\(\-[a-zA-Z-]\+\
 CURRENT_VERSION=$(yq eval '.version' ./package.json)
 
 if check_version_greater "$CURRENT_VERSION" "$PUBLISHED_VERSIONS"; then
-  echo "##[set-output name=VERSION;]v$CURRENT_VERSION"
-  echo "##[set-output name=BUILD_DATE;]$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
-  echo "##[set-output name=IS_NEW_VERSION;]true"
+  echo "VERSION=v$CURRENT_VERSION" >> $GITHUB_OUTPUT
```

Comment:
is this right, no idea what the `##[            ;]` in the original command is doing

Reply:
reminder of the change https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/